app/thermo.py

```diff
@@ -89,11 +88,11 @@ def avrg(self, readfile, writefile, debug=False):
         avrgtemp = round(numpy_average, 2)
         tup_temp_refined = (start, avrgtemp)
         fileRW.write("/home/pi/" + writefile, tup_temp_refined)
```

Comment:
For me I would like to get rid of all fixed paths in the code and have them only changed in one place hedge.py.  This may have to wait

Reply:
Will start work on this